e2e/__tests__/__snapshots__/failures.test.ts.snap

```diff
@@ -387,9 +387,9 @@ FAIL __tests__/snapshotNamed.test.js
 
   ● failing named snapshot
 
-    expect(value).toMatchSnapshot()
+    expect(received).toMatchSnapshot(hint)
 
-    Received value does not match stored snapshot "failing named snapshot: snapname 1".
+    Snapshot name: \`failing named snapshot: snapname 1\`
```

Comment:
Should probably make this something like `failing snapshot with hint: hint` in the test as well then so the naming is consistent everywhere?

Reply:
Yes, good catch!

* rename `e2e/failures/__tests__/snapshotNamed.test.js` as `snapshotWithHint.test.js`
* rename corresponding `.snap` file
* rename `'works with named snapshot failures'` as `'works with snapshot failures with hint'` in `failures.test.ts` but until I turned off an editor plugin, I made a temporary mess of a multiline template literal with spaces at ends of lines in `'works with node assert'` test :(

One other occurrence: rename `'handles property matchers with custom name'` as `'handles property matchers with hint'` in `toMatchSnapshot.test.ts`